### PR TITLE
domain for portfolio please approve

### PR DESCRIPTION
**Pull request by RAYXSTYDIOZ**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: 0.78.json
Comment (CuteDog5695) [inline 0.78.json:1]: 78.is-a.dev does not exist, so you cant make nested domains with it

File: 0.78.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "owner": {
+    "username": "<RAYAN GAMES>",
+    "email": "<kgold3796@gmail.com>"
+  }
+}
```
Comment (CuteDog5695) [inline 0.78.json:6]: username line is wrong, email line is wrong and there isnt even a record here

File: domains/sussyrayan-is.a-dev.json
Comment (CuteDog5695) [inline domains/sussyrayan-is.a-dev.json:1]: remove -is-a.dev from the .json file

File: domains/sussyrayan-is.a-dev.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "Portfolio website ",
+    "repo": "https://rayxstydioz.github.io/RAYXSTUDIOZ/",
+    "owner": {
+        "username": "RAYXSTYDIOZ",
+        "email": "syedyaseeralirayan@gmail.com"
+    },
+    "record": {
+        "CNAME": "rayxstydioz.github.io"
```
Comment (CuteDog5695) [inline domains/sussyrayan-is.a-dev.json:9]: site does not exist

**Review by iostpa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**